### PR TITLE
Add Homescreen, with searchfield component and randomised greeting 

### PR DESCRIPTION
**Pull request by me-jterhorst**
In this pull request 

I added the Homescreen component, 

A searchfield component with rounded lines, 
the greeting text changes onload to one of 7 alternatives, 
a rudimentary capability was added so that loggin status and user name can be used throughout the greeting component.

### CODE REVIEW

**Review by mpagels — COMMENTED** [review]
What is happening here? No pull request description. :-(
The Pull request title let's assume that some routing was happening in the code? But I see just an addition in CSS. Pls provide more guidance with a pull request description.
Thx

File: client/src/components/Greeting.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "what's cookin'?",
+    "what's happening?",
+    "what's the plan?",
+    "let's get preppin'?",
+    "what's on your remind?",
+    "got a new idea?",
+    "what's lined up?",
+  ];
+
+  const number = Math.floor(Math.random() * greetings.length);
```
Comment (patrikglanemann) [inline client/src/components/Greeting.js:14]: Cool idea to have different greetings and a simple, clever solution to display them randomly:)

File: client/src/components/Searchfield.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.searchfield {
+  position: relative;
+  margin: 0 auto;
+  height: 40px;
```
Comment (patrikglanemann) [inline client/src/components/Searchfield.css:4]: Would it be better to use rem here?

**Review by patrikglanemann — COMMENTED** [review]
LGTM
Your code and files are very good structured from the beginning.
I like the (nearly) already finished styling and the transition effect of the menu navigation!

File: client/src/App.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import "./App.css";
 import BottomNav from "./components/BottomNav";
 import Header from "./components/Header";
+import Home from "./pages/Home";
 import { Switch, Route, Redirect } from "react-router";
 
 export default function App() {
```
Comment (johnxgaehner) [inline client/src/App.js:7]: I can not comment somehow in the Switch section. You used the exact path in nearly every Route but it should also work without exact, right? I think you will move the main element inside the component like you've done in your Home component, right?

File: client/src/components/Searchfield.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  content: "";
+  position: absolute;
+  bottom: -5px;
+  height: 5px;
+  width: 100%;
+  background: var(--color-black-100);
+  border-radius: 5px;
+}
+
+.searchfield-button {
```
Comment (johnxgaehner) [inline client/src/components/Searchfield.css:28]: BEM would be Searchfield__button. :)
as searchfield is also your component, you capitalize it.

**Review by johnxgaehner — APPROVED** [review]
bam! love it.

File: client/src/components/Greeting.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <h1 className='greeting'>
```
Comment (namirsab) [inline client/src/components/Greeting.js:16]: This is just depending on which convention you want to use here. But normally i'll name the class of the root element of a component with the same name of the component (in this case, `"Greeting"`). Same for the other components

File: client/src/components/Searchfield.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import "./Searchfield.css";
+import { ReactComponent as SearchIcon } from "../svg/icon-search.svg";
+
+export default function Searchfield() {
```
Comment (namirsab) [inline client/src/components/Searchfield.js:4]: Impressive component and showing css skills! Very good!

File: client/src/index.css [hunk 7]
```diff
@@ -52,3 +52,8 @@ body {
   flex-direction: column;
   min-height: 100vh;
 }
+
+main {
+  padding-top: 125px;
+  margin-inline: 20px;
+}
```
Comment (namirsab) [inline client/src/index.css:59]: I'd use rem or em for padding and margin :)

**Review by namirsab — APPROVED** [review]
@me-jterhorst well done!
I only miss that you link this PR with a ticket ;) 

And i left a few comments, nothing big though. Take a look!

File: client/src/components/Searchfield.css (hunk 2 above)
Comment (me-jterhorst) [reply client/src/components/Searchfield.css:4]: actually yes :) thank you

File: client/src/components/Searchfield.js (hunk 6 above)
Comment (me-jterhorst) [reply client/src/components/Searchfield.js:4]: Thank you :)

**Pull request merged**